### DISCUSSION TITLE
SDK Floating Point Operations

### DISCUSSION THREAD

**Discussion by cosparks** [earlephilhower/arduino-pico#1453] [Q&A] [answered]
By default, does arduino-pico make use of the pico SDK implementation of single and double-precision floating point operations?  I didn't find anything in the documentation or in forum posts. I would assume that it does, but just want to *double* check.

Edit: For reference, documentation is in section 2.7.2 of the [pico sdk datasheet](https://datasheets.raspberrypi.com/pico/raspberry-pi-pico-c-sdk.pdf)

**Comment by earlephilhower** [comment, marked as answer]
Yes.  See the `lib/platform_wrap.txt` file.
